roles/openstack_helm_infra_openvswitch/defaults/main.yml

```
@@ -1,19 +1,20 @@
----
-# .. vim: foldmarker=[[[,]]]:foldmethod=marker
-
-# .. Copyright (C) 2022 VEXXHOST, Inc.
-# .. SPDX-License-Identifier: Apache-2.0
-
-# Default variables
-# =================
-
-# .. contents:: Sections
-#    :local:
-
-
-# .. envvar:: openstack_helm_infra_openvswitch_values [[[
+# Copyright (c) 2023 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
 #
-# Overrides for Helm chart values
-openstack_helm_infra_openvswitch_values: {}
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+openstack_helm_infra_openvswitch_helm_release_name: infra_openvswitch
+openstack_helm_infra_openvswitch_helm_chart_path: "{{ role_path }}/../../charts/infra_openvswitch/"
+openstack_helm_infra_openvswitch_helm_chart_ref: /usr/local/src/infra_openvswitch
```

Now helm chart is downloaded as name `openvswitch`. Is there any specific reason for this `infra_openvswitch` name? @guilhermesteinmuller 
```suggestion
openstack_helm_infra_openvswitch_helm_release_name: openvswitch
openstack_helm_infra_openvswitch_helm_chart_path: "{{ role_path }}/../../charts/openvswitch/"
openstack_helm_infra_openvswitch_helm_chart_ref: /usr/local/src/openvswitch
```